src/core/api/public_api.ts

```diff
@@ -1695,19 +1695,28 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     let trackId : string;
     let switchingMode : IAudioTrackSwitchingMode | undefined;
     let reprsToLock : string[] | null = null;
+    let relativeResumingPosition : number | undefined;
     if (typeof arg === "string") {
       trackId = arg;
     } else {
       trackId = arg.trackId;
       periodId = arg.periodId;
       switchingMode = arg.switchingMode;
       reprsToLock = arg.lockedRepresentations ?? null;
+      relativeResumingPosition = arg.relativeResumingPosition;
     }
     return this._priv_callTracksStoreGetterSetter(
       periodId,
       undefined,
       (tcm, periodRef) =>
-        tcm.setAudioTrack(periodRef, trackId, switchingMode, reprsToLock));
+        tcm.setAudioTrack({
+          periodObj: periodRef,
+          wantedId: trackId,
+          switchingMode,
+          reprsToLock,
```

Comment:
Now that those are property keys, I think it would be better to provide more intelligible names (e.g. `representationsId`)